rpcs3/Crypto/unpkg.h

```diff
@@ -297,11 +297,19 @@ struct PKGMetaData
 	} self_info;
 };
 
-enum class package_error
+struct package_install_result
 {
-	no_error,
-	app_version,
-	other
+	enum class error_type
+	{
+		no_error,
+		app_version,
+		other
+	} error;
+	struct version
+	{
+		std::string_view expected;
+		std::string_view found;
```

Comment:
Never store temporary data in string_view
```suggestion
		std::string expected;
		std::string found;
```